test/new_relic/agent/configuration/default_source_test.rb

```diff
@@ -291,6 +285,81 @@ def test_allowlist_blocks_invalid_values_and_uses_a_default
       end
     end
 
+    def test_boolean_configs_accepts_yes_on_and_true_as_strings
+      key = :'send_data_on_exit'
+      default = ::NewRelic::Agent::Configuration::DefaultSource.default_for(key)
```

Comment:
I don't see the `default` var being used.

Reply:
Yup yup - copy pasta error. Removed! https://github.com/newrelic/newrelic-ruby-agent/pull/2847/commits/520e9b5c421552188cdfcf4a02af0fd315076196